internal/cloud/realm/auth_test.go

```diff
@@ -78,16 +78,46 @@ func TestRealmAuthRefresh(t *testing.T) {
 		assert.Equal(t, nil, err)
 
 		// invalidate the session's tokens
-		session.AccessToken = ""
 		session.RefreshToken = session.AccessToken
+		session.AccessToken = ""
```

Comment:
Not super important, but I flipped these so both weren't set to "" (doesnt affect the result since we return invalid session regardless of the actual refresh error here)